bin/migrate-oats-data/submissions/sql/app_submission.sql

```diff
@@ -0,0 +1,26 @@
+WITH
```

Comment:
Just to clarify so, are OATS components equivalent to ALCS Submissions? And is it 100% overlap with imported applications?